lang/java.md

```diff
@@ -0,0 +1,114 @@
+Java installed, and more are created every day. Java is a general-purpose, class-based, object-oriented programming language designed for having lesser implementation dependencies. 
+It is a computing platform for application development. The compilation of the Java applications results in the bytecode that can be run on any platform using the 
+Java Virtual Machine(JVM). Java is fast, secure, and reliable. From laptops to datacenters, game consoles to scientific supercomputers, cell phones to the Internet, Java is 
+everywhere!
+
+
+## What is Java used for?
+
+**Here are some important Java applications:-**
+
+✦ It is used for developing Android Apps
+
+✦ Helps you to create Enterprise Software
+
+✦ Wide range of Mobile java Applications
+
+✦ Scientific Computing Applications
+
+✦ Use for Big Data Analytics
+
+✦ Java Programming of Hardware devices
+
+✦ Used for Server-Side Technologies like [Apache](https://commons.apache.org/), [JBoss](https://developers.redhat.com/products/eap/overview), [GlassFish](https://javaee.github.io/glassfish/), etc.
+
+    
+## Why to use Java?
+Java was designed to be easy to use and is therefore easy to write, compile, debug, and learn than other programming languages. Java is object-oriented. This allows you to 
+create modular programs and reusable code. Java is platform-independent.
+
+Java is a solid core language. Web and non-web environments, all operating systems, all types of devices you’ll find Java everywhere. For those in favor of learning Java 
+first, a key factor is that it teaches you to think like a programmer. You’ll learn the language, but you’ll also learn some key underpinnings of programming as well. It’s 
+also an object oriented programming (OOP) language. Learning OOP requires students to master a certain level of logic. You have to learn how to think through processes in both
+the specific and the abstract. Again, that provides a good base for learning programming in general.
+
+
+## Tools You Will Need
+
+For setting up environment for performing java codes you need with a minimum of 64 MB of RAM .
+
+You will also need the following softwares −
+- Linux 7.1 or Windows xp/7/8 operating system
+- Java JDK 8
+- Text Editors
+
+
+## Setting up Java development environment
+
+Java SE is freely available from the link [Download Java](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html). You can download a version based on your 
+operating system.Follow the instructions to download Java and run the .exe to install Java on your machine. Once you installed Java on your machine, you will need to set 
+environment variables to point to correct installation directories −
+
+**➤ Setting Up the Path for Windows :-**
+
+Assuming you have installed Java in *c:\Program Files\java\jdk* directory −
+
+- Right-click on 'My Computer' and select 'Properties'.
```

Comment:
`My Computer` is now `This PC` in modern versions of Windows. Please change to avoid confusion. Surround with backticks (`) instead of single quotes for better readablity.